base_multi_company/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 {
     'name': 'Multi Company Base',
     'summary': 'Provides a base for adding multi-company support to models.',
-    'version': '10.0.1.0.1',
+    'version': '10.0.1.0.2',
```

Comment:
Shouldnot this be 10.0.2.0.0 as there is a new constraint and migration script involved?

Reply:
done in https://github.com/OCA/multi-company/pull/83/commits/2ccf60c32e773bf1b34f4f37bc2f950093cfe874